ui/components/ui/multi-select.tsx

```
@@ -0,0 +1,1066 @@
+                                  'mr-2 h-4 w-4',
+                                  responsiveSettings.compactMode && 'mr-1 h-3 w-3',
+                                  customStyle?.iconColor && 'text-current',
+                                )}
+                                {...(customStyle?.iconColor && {
+                                  style: { color: customStyle.iconColor },
+                                })}
+                              />
+                            )}
+                            <span className={cn(screenSize === 'mobile' && 'truncate')}>{option.label}</span>
+                            <div
+                              role="button"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)